aperturedb/ParallelQuery.py

```diff
@@ -7,6 +8,73 @@
 logger = logging.getLogger(__name__)
 
 
+def execute_batch(q, blobs, db,
+                  response_handler: Callable = None, commands_per_query: int = 1, blobs_per_query: int = 0):
+    """
+    Execute a batch of queries, doing useful logging around it.
+    Calls the response handler if provided.
+    This should be used (without the parallel machinery) istead of db.query to keep the
+    response handling consistent, better logging, etc.
+
+    Returns:
+        - 0 : if all commands succeeded
+        - 1 : if there was -1 in the response
+        - 2 : For any other code.
+    """
+    result = 0
+    logger.debug(f"Query={q}")
+    r, b = db.query(q, blobs)
+    logger.debug(f"Response={r}")
+
+    if db.last_query_ok():
+        if response_handler is not None:
+            # We could potentially always call this handler function
+            # and let the user deal with the error cases.
+            blobs_returned = 0
+            for i in range(math.ceil(len(q) / commands_per_query)):
+                start = i * commands_per_query
+                end = start + commands_per_query
+                blobs_start = i * blobs_per_query
+                blobs_end = blobs_start + blobs_per_query
+
+                b_count = 0
+                for req, resp in zip(q[start:end], r[start:end]):
+                    for k in req:
+                        # Ref to https://docs.aperturedata.io/parameters/blobs.html
+                        blobs_where_default_true = \
+                            k in ["FindImage", "FindBlob", "FindBlobs"] and (
```

Comment:
```suggestion
                            k in ["FindImage", "FindBlob"] and (
```
There is no "FindBlobs" command.